src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -325,7 +327,7 @@ async def set_output_lut_parameters(
         # triggering is not supported by the TPZ device
         payload = st.pack(
             "<HHHLLLLHLH",
-            1,
+            _CHANNEL,
```

Comment:
```suggestion
            Tpz._CHANNEL,
```
Use class constants.